### PR TITLE
(Experimental) Support Wifi Scanning from 1 browser tab

### PR DESCRIPTION
**Pull request by thienhoang23**
2+ browser tabs will encounter the following problems:
- If they all press Refresh at the same time, then only 1 of them can go through, the other ones are going to fail due to that one going through hogging the up the WiFi scanning resources.
- If a tap updates its state after the refresh, nothing about this update is reflected on the other tabs. That is, there will be some inconsistencies between the tabs.

However, for a one-user system, there aren't any use cases where a user needs more than 1 tab to connect to the WiFi. If he/she is not most up to date to the state of the server, they can just hit Refresh.

### CODE REVIEW

File: cmds/wifi/wifiServer.go [hunk 1]
```diff
@@ -43,29 +42,41 @@ function replaceWithConnecting(elem) {
...
+	fetch("http://localhost:8080/refresh").then(_ => window.location.reload())
+}
+
+function disableOtherButton(elem) {
     btns = document.getElementsByClassName("btn");
-    var i;
+	var i;
     for (i = 0; i < btns.length; i++) {
     	if (btns[i] === elem) {
     		continue;
```
Comment (erfanian) [inline cmds/wifi/wifiServer.go]: Consider https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of if possible, it is less errorprone.

File: cmds/wifi/wifi.go [hunk 2]
```diff
@@ -53,6 +57,19 @@ func init() {
 	}
 }
 
+func scanWifi() {
+	o, err := exec.Command("iwlist", *iface, "scanning").CombinedOutput()
```
Comment (erfanian) [inline cmds/wifi/wifi.go]: *iface

Never supplied as an input from the web interface, right?

**Comment by erfanian** [comment]
Probably want to wait on another project owner's approval?

File: cmds/wifi/wifi.go (hunk 2 above)
Comment (thienhoang23) [reply cmds/wifi/wifi.go]: Nope, only from the cmd

**Comment by thienhoang23** [comment]
I'll wait for @rminnich or @hugelgupf to give a LGTM then I'll merge

File: cmds/wifi/wifiServer.go (hunk 1 above)
Comment (thienhoang23) [reply cmds/wifi/wifiServer.go]: ok, I'll update this

**Pull request merged**